libs/upd/views/projects/src/lib/projects-home/projects-home.component.ts

```diff
@@ -47,6 +47,8 @@ export class ProjectsHomeComponent implements OnInit {
             header: this.i18n.service.translate('Name', lang),
             type: 'link',
             typeParam: '_id',
+            displayFilterOptions: false,
+            displayTable: true,
```

Comment:
Seeing all these `displayTable: true` makes me realize that it would probably be better for it to be `hideTable` and the logic be swapped, that way it's displayed by default, which I think kind of makes way more sense.

Reply:
I was thinking that during the creation, but, since there are columns I don't want in the tree select (i.e. Tasks, Visits, Calls), I decided to use both. BUT! I added it as suggested.